packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -44,12 +46,12 @@ class _TraceCheckBoxColumn extends ColumnData<TracedClass>
   }) {
     return Checkbox(
       value: item.traceAllocations,
-      onChanged: (value) async {
+      onChanged: (value) {
         ga.select(
           gac.memory,
           '${gac.MemoryEvent.tracingTraceCheck}-$value',
         );
-        await controller.setAllocationTracingForClass(item.cls, value!);
+        unawaited(controller.setAllocationTracingForClass(item.cls, value!));
```

Comment:
is this comma related?

Reply:
oops. reverted. I really need to turn on auto-save in VSCode. That was supposed to be for a separate CL but I must have left the file unsaved and then hit save-all.